plugins/API/tests/System/AutoSuggestAPITest.php

```diff
@@ -293,11 +306,4 @@ public static function getPathToTestDirectory()
     }
 }
 
-$date = mktime(0, 0, 0, 1, 1, 2018);
-
-$lookBack = ceil((time() - $date) / 86400);
-
-API::$_autoSuggestLookBack = $lookBack;
-
 \Piwik\Plugins\API\tests\System\AutoSuggestAPITest::$fixture = new ManyVisitsWithGeoIPAndEcommerce();
```

Comment:
Would you know if it's possible to move the fixture initialisation into the `setUpBeforeClass` method as well?

Reply:
you'd have to do it before `parent::setUpBeforeClass`()`.

Reply:
That is where we are moving the rest of the code already, or do you mean somewhere else?

I went and had a look how other tests do it and it's a common occurrence to set up the fixture outside of the test class. A bit unconventional, but the change here would be out of line with the rest, so we can look at moving the fixture creating to `setUpBeforeClass` before calling `parent::setUpBeforeClass` globally in a separate ticket.

Reply:
I am moving that code so it is only run when the AutoSuggestAPITest is run instead of every time it is included, and so it can be undone during test teardown. None of that applies to the `$fixture` variable. Your comment is irrelevant and you are wasting my time and energy. If the position of the static variable initialization bothers you so much, then by all means, spend your own personal time creating a PR.